### ISSUE TITLE
Cant have lua running in two host threads at a time

### ISSUE THREAD

**Issue by arcane360** [brossentia/BizHawk-Shuffler#16]
When I load the script, the shuffler starts and then gives an Operation Exception when the game switches to the other Roms,  I have to hit OK button to continue to the game, however this ruins the shuffler experience.  Do you know of a way to fix?
![ErrorBizHawk](https://user-images.githubusercontent.com/24868490/112790893-5c844600-902e-11eb-87c9-1a16b1b89557.png)

**Comment by JustinTime82** [comment]
I have the same problem, have no idea what to do either.

**Comment by Tommyhawk18** [comment]
Had this problem. I was using the latest version of biz hawk. Downloaded the one recommended in the shuffler read me. Made sure to delete not just the current and compete rom deleteme but also a third in another folder. Also I had moved the desktop.ini to complete by accident in the shuffler setting program so I loved it back. Not sure what the fix was but it stopped happening. Could be any of the three

**Comment by JustinTime82** [comment]
Yes I figured out the issue as well. I used version 2.5.3 of bizhawk with the randomizer and it seems to work just fine. I was previously using the newer version of bizhawk and the error kept popping up as the rom switched. So I'm guessing they need to update the randomizer to work with the newer version. :)

**Comment by tonylukasavage** [comment]
I've got my own bizhawk/lua project going now and happened upon this issue. Still seeing this behavior as of 2.6.2, just as a heads up. It appears to be a bug they fixed in 2.5.2 but somehow got reintroduced. I opened a new issue there referencing the old bug, so hopefully they'll fix it in newer versions. Feel free to follow along here: https://github.com/TASVideos/BizHawk/issues/2940
